include/cinder/Log.h

```diff
@@ -135,18 +140,46 @@ class LoggerBreakpoint : public Logger {
 	Level	mTriggerLevel;
 };
 
-#if defined( CINDER_COCOA )
-
-class LoggerSysLog : public Logger {
+class LoggerSystem : public Logger {
 public:
-	LoggerSysLog();
-	virtual ~LoggerSysLog();
-
-	void write( const Metadata &meta, const std::string &text ) override;
-};
-
+	LoggerSystem();
+	virtual ~LoggerSystem();
+	
+	void write( const Metadata &meta, const std::string &text ) override { mImpl->write( meta, text ); }
+	// SysLog and EventLog logging levels inherently work differently (see specific comments).  Currently
+	// this doesn't matter since logging implimentation only allows once instance of a logger type.
```

Comment:
The github diff may be confusing since it doesn't show that my logging level comments pertain to the function below
```
// SysLog and EventLog logging levels inherently work differently (see specific comments).  Currently
// this doesn't matter since logging implimentation only allows once instance of a logger type.
void setLoggingLevel( Level minLevel ) { mImpl->setLoggingLevel( minLevel ); }
```

To elaborate on how they are different:

In **SysLog** changing the logging level is done through a call to ```setlogmask```.  (I need to make it clear this is a system level call ie ```::setlogmask```).  This causes the OS X system logger to mask the logging call.  This would be applied to all SysLog calls from the application.

In **EventLog** we keep an internal log level and use an if statement to decide if we should log.

If, in future versions of the logger we allowed multiple ```LoggerEventLogs``` or multiple ```LoggerSysLogs```, then changing the logging level on one single SysLog would affect all SysLogs.  This is not the case for EventLog.  Also, if for some reason the user is calling syslog elsewhere in the app (outside of our CI_LOG), then the mask would also be applied to their syslog calls.

I could not use the setlogmask call, and do the same if statement check when logging either type.  It may actually be more efficient than passing our log message to SysLog and letting SysLog filter.  In that case both implementations would be functioning the same.

After writing all of this I'm leaning towards doing the loglevel check in ```LoggerSystem``` rather than in either ```LoggerSysLog``` or ```LoggerEventLog``` implementation.